src/utils/webrtc/simplewebrtc/simplewebrtc.js

```diff
@@ -121,7 +121,7 @@ function SimpleWebRTC(opts) {
 			}
 		} else if (peers.length) {
 			peers.forEach(function(peer) {
-				if (message.sid) {
+				if (message.sid && !self.connection.hasFeature('mcu')) {
```

Comment:
but now it still (always) runs `peer.handleMessage(message)` or is that what was missing?

Reply:
Now `peer.handleMessage(message)` is always run; before `peer.handleMessage(message)` was not run when the HPB was used due to the `peer.sid === message.sid` check. For the full explanation please refer to the message of the first commit ;-)